bears/xml/XMLBear.py

```diff
@@ -1,5 +1,26 @@
+import os
+import requests
+
 from coalib.bearlib.abstractions.Lint import Lint
 from coalib.bears.LocalBear import LocalBear
+from coalib.misc.Shell import escape_path_argument
+from coalib.settings.Setting import path
+
+
+def path_or_url(type1, type2):
+    '''
+    Checks if the argument is a ``url`` or a ``path``.
+
+    :param type1:       path
+    :param type2:       str
+    :return:            path or str
+    '''
+    def _path_or_url(dtdvalid):
+        if dtdvalid is not None and os.path.exists(dtdvalid):
+            return type1
```

Comment:
@AbdealiJK I changed it to type1(dtdvalid) still it is giving the same error while running tests
```E           AssertionError: 0 == 0 : The local bear 'XMLBear' yields no result although it should. 
E           Running bear XMLBear...
E           The bear XMLBear cannot be executed. Unable to convert parameter 'dtdvalid' into type <function path_or_url.<locals>._path_or_url at 0x7f5a776421e0>```